README.md

```diff
@@ -137,6 +137,7 @@ line_numbers: true
 index: false
 toc: Table of Contents
 theme: _theme
+open: false
```

Comment:
All the command line options have the same name in the config.
The only difference between the config name and CLI name is predictable:

- hyphens in CLI become underscore in config.
- in case where the config default is `true`, the CLI flag starts with `--no-*`.